### PR TITLE
Add a working example

### PR DESCRIPTION
**Pull request by thomashilke**
The goal is to add a complete working example to the readme, to help an interested user to start off.

### CODE REVIEW

**Review by Admir-H — APPROVED** [review]

**Review by TimotheePouchon — COMMENTED** [review]
Cool example. Can't wait for the graphic app :-D

File: README.md [hunk 1]
```diff
@@ -114,6 +116,103 @@ calculator to the constructor of all these object, a calculator is
 instantiated in the factory, and object instances are created through
```
Comment (TimotheePouchon) [inline README.md:116]: To alleviate the need to pass an instance of the
calculator to the constructor of all these **objects**, a calculator is
instantiated in the factory, and object instances are created through
the factory's **methods** `CreatePoint`, `CreateSegment`, etc.

File: README.md [hunk 2]
```diff
@@ -114,6 +116,103 @@ calculator to the constructor of all these object, a calculator is
...
 the factory's methodes `CreatePoint`, `CreateSegment`, etc.
 
+# Working example
+The following example demonstrate the library usage to compute the
```
Comment (TimotheePouchon) [inline README.md]: demonstrates

**Review by TimotheePouchon — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (thomashilke) [reply README.md]: C'est corrige!

File: README.md (hunk 1 above)
Comment (thomashilke) [reply README.md:116]: `methods` c'est deja correct, non?

Comment (thomashilke) [reply README.md:116]: C'est corrige, sinon.

Comment (TimotheePouchon) [reply README.md:116]: Ben non c'est écrit `methodEs`... :-D

**Review by TimotheePouchon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (thomashilke) [reply README.md:116]: Ahhh oue! j'etais focalise sur le `s`. C'est bon.

**Pull request merged**